js/classes/tagmodel.js

```diff
@@ -6,10 +6,10 @@ class TagModel {
     this.openDocs = [];
     this.currentCategory = null;
     this.categories = [];
+    this.currentModel = null;
```

Comment:
This probably shouldn't default to null, since it causes an error trying to join the file paths.  Maybe set to "", or set to a default "untitled" or something, so that the path join works, unless you named it somewhere else